components/config-provider/__tests__/__snapshots__/components.test.js.snap

```diff
@@ -10850,7 +10850,7 @@ exports[`ConfigProvider components Slider configProvider 1`] = `
   />
   <div>
     <div
-      class="config-tooltip config-slider-tooltip"
+      class="config-tooltip"
```

Comment:
Miss prefix className, should check why.

Reply:
will be resolved after updating snapshots